src/responsibleai/rai_analyse/rai_component_utilities.py

```diff
@@ -155,7 +159,78 @@ def load_mlflow_model(
         _logger.info("Successfully installed model dependencies")
 
     try:
-        model = mlflow.pyfunc.load_model(model_uri)._model_impl
+        if not use_separate_conda_env:
+            model = mlflow.pyfunc.load_model(model_uri)._model_impl
+            return model
+
+        # Serve model from separate conda env using mlflow
+        mlflow_models_serve_logfile_name = "mlflow_models_serve.log"
+        try:
+            # run mlflow model server in background
+            with open(mlflow_models_serve_logfile_name, "w") as logfile:
```

Comment:
I'm a bit worried as I've used this before and it's incredibly difficult to debug, the debug logs are very difficult to find and understand

Reply:
I share your concern. The choices I had were
- write to a file OR
- don't capture at all

Given this consideration I chose option #1 😄